release/platform/platform.go

```diff
@@ -388,6 +388,14 @@ var platforms = []Platform{
 		Repos:     []Repo{RepoEnterprise, RepoOrg},
 		BuildTags: defaultBuildTags,
 	},
+	{
+		Name:      "debian12",
+		Arch:      ArchX86_64,
+		OS:        OSLinux,
+		Pkg:       PkgDeb,
+		Repos:     []Repo{RepoEnterprise, RepoOrg},
```

Comment:
Seems like we do if there is RepoOrg here?

Reply:
Yes, added.